src/fmu/sumo/sim2sumo/common.py

```diff
@@ -327,20 +234,16 @@ def __init__(
     ):
         self._logger = logging.getLogger(__name__ + ".Dispatcher")
         self._limit_percent = 0.5
-        self._parentid = get_case_uuid(datafile)
+        self._parentid = get_case_uuid(Path.cwd(), parent_level=1)
```

Comment:
This is breaking `tests/test_functions.py::test_Dispatcher`.
The path, at least in the test, no longer points to the testfile.

Reply:
Test passes after latest changes 👍